docker/README.md

```diff
@@ -1,16 +1,26 @@
-# Creating docker containers for snapcraft
+# Docker images for `snapcraft`
 
-By default the `Dockerfile` builds Ubuntu 16.04 (Xenial) image with `snapcraft` from the `edge` channel.
+OCI-compliant container images and their sources are officially supported by
```

Comment:
@tigarmo how "official" are the snapcraft-rocks?

Reply:
@lengau it is being the repo of verified organization https://github.com/canonical. Is that not enough?

Or maybe you're also confused by  "maintained by the Starcraft team"? )

![image](https://github.com/user-attachments/assets/9e082f62-2862-4150-a462-f95b89d16375)

The screenshot of what Google and Blizzard StartCraft team were doing for AI in 2016. )

Reply:
> @tigarmo how "official" are the snapcraft-rocks?

Well it's the way the Starcraft team decided to support Snapcraft images, so as official as it gets I guess. I've been slacking on supporting the rocks but that's a separate issue.